src/triage/component/catwalk/exceptions.py

```diff
@@ -0,0 +1,13 @@
+
+__all__ = ['BaselineFeatureNotInMatrix']
+
+
+class BaselineFeatureNotInMatrix(Exception):
+    """ This error is used to allow feature mixing and baseline classes to be
+    included in the same experiment.
+
+    Without error handling, the baseline classes would cause the experiment to
+    end prematurely when they received a matrix without the required feature
+    (if, for example, leave-one-out feature mixing is enabled). Raising this
+    error will cause the model to be skipped elegantly.
+    """
```

Comment:
👍 👍 

Perhaps debatable, but now that you've defined this fancy app-specific exception, you might _also_ have it derive from `KeyError`, since that's also essentially what it is:

    class BaselineFeatureNotInMatrix(KeyError):
        …